src/cmpClient.c

```diff
@@ -132,8 +131,13 @@ bool opt_disable_confirm;
 const char *opt_certout;
 const char *opt_chainout;
 
+/* certificate enrollment and revocation */
```

Comment:
Well, either leave out this comment
or add also the related ones contained in `cmpossl/apps/cmp.c`.